src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incentives/IncentivesMigrationIntTest.kt

```diff
@@ -100,7 +100,7 @@ class IncentivesMigrationIntTest : SqsIntegrationTestBase() {
 
       await atMost Duration.ofSeconds(60) untilAsserted {
         verify(telemetryClient).trackEvent(
-          eq("nomis-migration-incentives-completed"),
+          eq("nomis-migration-completed"),
           any(),
```

Comment:
improve custom event names and switch to reflection approach for bean properties for telemetry